package.json

```diff
@@ -83,10 +84,11 @@
   "scripts": {
     "start": "npm run storybook:run",
     "commit": "git-cz",
-    "build": "npm run build:scripts && npm run build:sass && npm run build:less",
+    "build": "npm run build:scripts && npm run build:sass && npm run build:css && npm run build:less",
     "build:scripts": "babel src --out-dir dist/js --ignore test*.js,__mocks__",
     "build:less": "mkdir -p dist/less && cp -r less/* dist/less",
     "build:sass": "mkdir -p dist/sass && cp -r sass/patternfly-react/* dist/sass && node-sass --output-style compressed --include-path sass $npm_package_sassIncludes_patternfly $npm_package_sassIncludes_bootstrap $npm_package_sassIncludes_fontAwesome -o dist/css sass/patternfly-react.scss",
+    "build:css": "mkdir -p dist/css && find ./src -name \\*.css -exec cp {} dist/css/ \\;",
```

Comment:
we shouldn't have css in the components directories. We've standardized on both less and sass so that we can use variables which can then be overridden.

Reply:
ok, done